src/Controls/tests/TestCases.HostApp/Issues/Issue26662.xaml.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 26662, "Unable to dynamically set unselected IconImageSource Color on Android", PlatformAffected.All)]
+	public partial class Issue26662 : TabbedPage
+	{
+		public Issue26662()
+		{
+			InitializeComponent();
+		}
+
+	}
+
+	public class Issue26662Tab1 : ContentPage
+	{
+		public Issue26662Tab1()
+		{
+			//If no FontImageSource color is given, tab icon should be applied based on the SelectedTabColor and UnselectedTabColor.
+			IconImageSource = new FontImageSource
+			{
+				FontFamily = "Ion",
+				Glyph = "\uf30c",
+				Size = 15
+			};
+			Button button = new Button
+			{
+				Text = "Change Icon Color",
+				BackgroundColor = Colors.Blue,
+				TextColor = Colors.Red
+			};
+			button.AutomationId = "Button";
+			button.Clicked += (s, e) =>
+			{
+				//If the FontImageSource color is given, the tab icon color should be applied solely based on the specified color, regardless of the SelectedTabColor or UnselectedTabColor.
+				IconImageSource = new FontImageSource
+				{
+					FontFamily = "Ion",
+					Glyph = "\uf30c",
+					Color = Colors.Violet,
```

Comment:
Can we add a test that only changes the Color on the FontImageSource and verify that that will update the color?

Reply:
Hi @PureWeen ,

Changing the `Color` property directly in `IconImageSource` does not trigger notifications at the Source level.

In TabbedPageManager on Android, `Page.PropertyChanged `only detects changes to IconImageSource, and this behavior is consistent across all platforms.

Currently, FontImageSource property changes are not handled in the TabbedPage implementation across platforms. We may need to add support for these changes to ensure proper updates.

Additionally, the user is following the same approach used in our test sample.

Reply:
It should trigger on the Source Level

changing the color triggers a source changed event

![image](https://github.com/user-attachments/assets/e19439a0-fc63-4254-9fd1-fdeb94ea8b94)

which fires a source changed event
![image](https://github.com/user-attachments/assets/cffed497-6406-4947-a8fd-0fc1c3718614)

It seems like we're just doing a bad job of listening for that event? 

I don't think we need to block this PR on this but it seems like we should be watching for the SourceChanged event more places and then reacting based on that. Can you create an issue?

Reply:
Hi @PureWeen ,
I meant the previous comment, as no implementation was added while changing the image source color property. As you mentioned, I have created a separate issue #[27549](https://github.com/dotnet/maui/issues/27549)  for this

Reply:
That PR was merged, do we have a test in this PR yet?